build/Dockerfile

```diff
@@ -9,5 +9,6 @@ RUN dnf install -b -y dnf-plugins-core && \
 
 COPY build/_output/bin/kubernetes-nmstate /usr/local/bin/kubernetes-nmstate
 COPY build/bin /usr/local/bin
+COPY bindata /bindata
```

Comment:
Let's forget about Docker for now, make template accessible for unit test, and let's just work the unit test.
After we are happy with Reconcile logic at unit test level, we start to wire things up.